sensor_msgs/CompressedImage mask
bool appear